flexget/plugins/sites/horriblesubs.py

```diff
@@ -27,7 +27,7 @@ def horrible_entries(requests, page_url):
             soup = get_soup(requests.get(page_url).content)
             log.debug('requesting: %s', page_url)
         except RequestException as e:
-            log.error('HorribleSubs request failed: %s', e)
```

Comment:
Should keep this logging since you're returning instead of raising an exception :P